blogs/2016/03/11/ExtensionsRoundup.md

```diff
@@ -0,0 +1,80 @@
+---
+Order: 4
+TOCTitle: Extensions Roundup
+PageTitle: Visual Studio Code Extensions
+MetaDescription: New, useful, and interesting extensions in Visual Studio Code. 
+Date: 2016-03-11
+ShortDescription: New, useful, and interesting extensions in Visual Studio Code.
+Author: Wade Anderson
+---
+
+# VS Code Extensions
+
+Three months have passed since we open sourced VS Code and introduced the [extensions API](http://code.visualstudio.com/docs/extensionAPI/vscode-api). 
+The [Extensions Marketplace](https://marketplace.visualstudio.com/VSCode) has over 850 extensions. Many new languages and frameworks are now supported.
+
+We will continue to iterate on improving the API and we would love your [feedback](https://github.com/Microsoft/vscode/issues). If extensions are new to 
+you, check out the [docs](http://code.visualstudio.com/docs/editor/extension-gallery) for instructions on finding and installing new extensions. We are invested
+in empowering the community to make VS Code the world's greatest code editor.
+
+In the spirit of one of our [favorite curated extensions list](https://github.com/viatsko/awesome-vscode), we want to give you regular updates on what is happening in the [VS Code Extensions Marketplace](https://marketplace.visualstudio.com/VSCode). 
```

Comment:
..one of our [favorite curated extensions lists](), we wanted to ...  
note: plural "lists" above and "we wanted to"
You could also try:
"In the spirit of a great [curated VS Code extensions list], we wanted to ..."